### DISCUSSION TITLE
Downgrading Issue

### DISCUSSION THREAD

**Discussion by CALR3F** [3kh0/ext-remover#516] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Esc Power Refresh doesn't do anything. I have tried doing when the chromebook was off and on, I have checked and all tthe keys work perfectly. My school disctrict probably somehow disabled it. Any way to fix this?

**Comment by obkduishinijf** [comment, marked as answer]
they can't disable or block it, make sure everything's correct like 3 or 4 times.

**Comment by funkymokey** [comment]
They literally cant block it
